tcs-service/src/main/java/com/transformuk/hee/tis/tcs/service/api/validation/PostValidator.java

```diff
@@ -193,6 +198,15 @@ private void notExistsFieldErrors(List<FieldError> fieldErrors,
     return fieldErrors;
   }
 
+  private void checkSubspecialtyIsOfTypeSubspecialty(List<FieldError> fieldErrors,
```

Comment:
Perhaps just personal preference, but I would name this 'checkSpecialtyIsOfTypeSubspecialty' since you could pass in any sort of specialty, not just subspecialties. The logic in the call on line 192 could make it clear that a subspecialty was being tested by renaming 's' as 'subspecialty' or similar).

Reply:
Agreed. The method takes in any specialty and checks whether it's of `specialtyType.SUB_SPECIALTY` or not. I've renamed it accordingly.
But the `s ->` on line 192 is indeed a parameter of that method, and can be all sorts of specialties, not just a subspecialty, so I renamed it to `specialty`